### ISSUE TITLE
Nested class: Can not assign to meta?

### ISSUE THREAD

**Issue by IngwiePhoenix** [marcobambini/gravity#255]
Hello! I tried Gravity and took it for a spin, especially since the docs said, it would have nested classes - which I really like. So, I made this:

```
class cA {
  class cB {
    func init() { System.print("B::init"); }
    func initB() { System.print("B::initB"); }
  }
  func init() { System.print("A::init"); }
}

func main() {
  var a = cA();
  var b = a.cB(); 
  b.initB();
}
```

But, this happens:
```
$ ./gravity test.g
A::init
B::init
B::initB
RESULT: NULL (in 0.0595 ms)
```

Now, from the code supplied above, this would be expected. However, if you change `class cB` into `static class cB` it frankly no longer exists:
```
 $ ./gravity test.g
A::init
RUNTIME ERROR: Unable to find cB into class cA
```

Neither can you achieve this with `public`. Basically, I can not seem to get this right... Oh, also remember to change `var b = a.cB()` to `var b=cA.cB()`.

Basically, I want to end up being able to structure a subsequent amount of classes similiar to a namespace. Example:
```
@class A
| @class B
| | @class C
// ...
var x = A.B.C();

// Or, with names maybe?
var e = App.Errors.InvalidCLIArg(...);
```

**Comment by marcobambini** [comment]
Hi @IngwiePhoenix, there are several unit test about static class declarations and all worked fine.
You can find them inside test/unittest.
For example:
- class_declaration.gravity
- class_static_nonstatic.gravity
- class_static.gravity
- enum_static.gravity
- fibonacci_class.gravity
- static_init.gravity
- static_init2.gravity
- string_interpolation.gravity
- test_classes.gravity
- test_declarations.gravity

I tried with the code:
```swift
class cA {
    static class cB {
        func init() { System.print("B::init"); }
        func initB() { System.print("B::initB"); }
    }
    func init() { System.print("A::init"); }
}

func main() {
    var a = cA.cB;
    return a;
}
```
and the result is:
`(CLASS) cB (0x10301a910) (in 0.0052 ms)`
which is fine.

When I try what you suggested:
```swift
func main() {
    var a = cA.cB.initB();
    return a;
}
```
I get the `RUNTIME ERROR: Unable to find initB into class cB meta`
and it makes perfectly sense to me because you are trying to access an instance method from a static class.

What you would need to do in order to be able to access initB from static class cB is to use the static keyword to declare that function static. The following code is the right to use:
```swift
class cA {
    static class cB {
        func init() { System.print("B::init"); }
        static func initB() { System.print("B::initB"); }
    }
    func init() { System.print("A::init"); }
}

func main() {
    var a = cA.cB.initB();
    return a;
}
```
